tests/authentication/test_models.py

```diff
@@ -122,7 +122,7 @@ def test_string_representation_is_correct_for_a_profile_model(self):
         self.assertEqual(str(self.profile1), "user@email.com's Profile")
 
     def test_that_user_profile_relationship_is_accurate(self):
-        self.assertEqual(self.user1.userprofile, self.profile1)
+        self.assertEqual(self.user1.profile, self.profile1)
 
```

Comment:
Since `userprofile` has been previously used, it is better we stick with that so we are certain we do not break the code anywhere.

Reply:
@Victoradukwu I have rectified this